src/modules/endpoint/endpoints/extras.js

```diff
@@ -8,6 +8,12 @@ import ParamMissingError from '../errors/ParamMissingError'
 
 const EXTRAS_ENDPOINT_NAME = 'extras'
 
+const createPostMap = (jsonPost: any): Map<string, string> => {
```

Comment:
`any` should not be used if we really need such a broad type here `mixed` is the better option. Doesn't linting show a warning for "unexpected use of weak type" here?

Reply:
Yes, I don't like it either, but we currently use `any` instead of `mixed` in all endpoints, so I wanted to stay consistent here... You can create a task for adjusting it, if you'd like :)